cfgov/apache/conf.d/headers.conf

```diff
@@ -10,6 +10,12 @@ Header always set X-Frame-Options SAMEORIGIN
 Header always set X-XSS-Protection: "1; mode=block"
 Header always set X-Content-Type-Options: nosniff
 
+# Add X-Forwarded headers when APACHE_HTTPS_FORWARDED_HOST envvar is set
+<If "-n osenv('APACHE_HTTPS_FORWARDED_HOST')">
+        RequestHeader set X-Forwarded-Proto https
+        RequestHeader set X-Forwarded-Host "${APACHE_HTTPS_FORWARDED_HOST}"
```

Comment:
Good question.  I'll dig into that and get back to you on that one.

Reply:
I believe this works as-is because it is injecting the `${APACHE_HTTPS_FORWARDED_HOSTS}` via [HTTPD config substitution](https://httpd.apache.org/docs/current/configuring.html#syntax), while the previous version used [`RequestHeader`](https://httpd.apache.org/docs/current/mod/mod_headers.html#requestheader)'s [ap_expr](https://httpd.apache.org/docs/current/expr.html) expression parser.  The latter is much more powerful, but since we only need to inject an envvar, this way seems simpler.